### ISSUE TITLE
No icon for DropBox

### ISSUE THREAD

**Issue by cgueret** [clearlinux/distribution#1699]
Step to reproduce:
* Install dropbox
* Install any of the "legacy icon" extension for gnome-shell (topicon, topicon redux, appnotifier, ...)
* Ensure dropbox is active

Expected result:
* See the icon of dropbox somewhere on the screen

Observed:
* No dropbox icon anywhere

**Comment by 0xA1B2** [comment]
Not going to work on CL (since ever). Only the command line Dropbox edition works.

**Comment by ahkok** [comment]
Since we don't support dropbox, this is technically not a bug - relabeling.

**Comment by cgueret** [comment]
Fair enough :) that said, I don't know if it's an issue with Dropbox or if any other app using a tray icon would face the same pb. Could you suggest a different app I could install to test?

**Comment by ahkok** [comment]
Gnome is moving away from showing tray icons by default for applications, so it's becoming more and more difficult to test this - apps are only supposed to show something when an event happens. I really can't think of any application right now :o

**Comment by cgueret** [comment]
Humm... OK :/ actually, if nautilus-Dropbox could be added as a package I'll be fine (I raised another ticket for it). All I need is to know if Dropbox is  running. Happy to try making that package too but I could not find the relevant documentation...

**Comment by lebensterben** [comment]
>Humm... OK :/ actually, if nautilus-Dropbox could be added as a package I'll be fine (I raised another ticket for it). All I need is to know if Dropbox is  running. Happy to try making that package too but I could not find the relevant documentation... 

@cgueret nautilus-Dropbox is proprietary... So I'm afraid they can not add that

**Comment by cgueret** [comment]
@lebensterben thanks for the precision ! What's then the best approach to install something that is not and won't be packaged? A good old make & & make install? :)

**Comment by lebensterben** [comment]
>@lebensterben thanks for the precision ! What's then the best approach to install something that is not and won't be packaged? A good old make & & make install? :) 

@cgueret nautilus-Dropbox is very easy to install and it can update itself in command-line.
For proprietary softwares I build them locally. But we may see 3rd party repo supported by swupd in this year, and then these software may be available there.
